_src/static/sass/components/_buttons.scss

```diff
@@ -0,0 +1,371 @@
+// #GLOBALS
+// #BASE (size)
+// #LARGE (size)
+// #SMALL (size)
+// #DEFAULT (style)
+// #DEFAULT-ALT (style)
+// #PRIMARY (style)
+// #SECONDARY (style)
+// #SECONDARY-ALT (style)
+// #LINK (style)
+// #ELEVATED (type)
+// #COMBO (type)
+// #ICONS (type)
+// #OVERLAY (type)
+// ----------------------------
+
+// ----------------------------
+// #LAYOUT
+// ----------------------------
+.example-set {
+    @extend %clear-last-child;
+    padding-bottom: spacing-vertical(base);
+    border-bottom: spacing-vertical(xx-small) solid palette(grayscale, xx-light);
+    margin-bottom: spacing-vertical(base);
+
+    .button-overlay-demo {
+        padding: 2rem;
+        background: palette(grayscale, base);
+    }
+}
+
+
+// ----------------------------
+// #GLOBALS
+// ----------------------------
+.btn {
+    @include transition(
+        color timing(xx-fast) ease-in-out 0s,
+        border-color timing(xx-fast) ease-in-out 0s,
+        background timing(xx-fast) ease-in-out 0s,
+        box-shadow timing(xx-fast) ease-in-out 0s
+    );
+    display: inline-block;
+    border-style: solid;
+    border-radius: 3px;
+    border-width: 1px;
+}
+
+// ----------------------------
+// #BASE
+// ----------------------------
+.btn-base {
+    font-size: font-size(mid-large);
+    padding: .8rem 1.2rem;
+}
+
+
+// ----------------------------
+// #LARGE
+// ----------------------------
+.btn-large {
+    font-size: font-size(large);
+    padding: 1rem 1.4rem;
+}
+
+
+// ----------------------------
+// #SMALL
+// ----------------------------
+.btn-small {
+    font-size: font-size(small);
+    padding: .6rem 1rem;
+}
+
+
+// ----------------------------
+// #DEFAULT
+// ----------------------------
+.btn-default {
+    background: palette(grayscale, white);
+    border-color: palette(primary, mid-dark);
+    color: palette(primary, mid-dark);
+
+    &:hover,
+    &.is-hovered,
+    &:focus,
+    &.is-focused {
+        background: palette(primary, mid-dark);
+        color: palette(grayscale, white);
+    }
+
+    &:active,
+    &.is-pressed,
+    &.is-active {
+        background: palette(primary, dark);
+        border-color: palette(primary, dark);
+    }
+
+    &:disabled,
+    &.is-disabled {
+        background: palette(grayscale, white);
+        border-color: palette(grayscale, x-light);
+        color: palette(grayscale, base);
+    }
+}
+
+
+// ----------------------------
+// #DEFAULT-ALT
+// ----------------------------
+.btn-default-alt {
+    background: $transparent;
+    border-color: palette(grayscale-cool, base);
+    color: palette(grayscale-cool, base);
+
+    &:hover,
+    &.is-hovered,
+    &:focus,
+    &.is-focused {
+        background: palette(primary, mid-dark);
+        border-color: palette(primary, mid-dark);
+        color: palette(grayscale, white);
+    }
+
+    &:active,
+    &.is-pressed,
+    &.is-active {
+        background: palette(primary, dark);
+        border-color: palette(primary, dark);
+    }
+
+    &:disabled,
```

Comment:
If we're using proper, semantic markup, adding disabled to the button (because we should be using a button for buttony things) should be enough. Can you give me an example of where this might be beneficial?